### ISSUE TITLE
Make spelling suggestions when renaming a file

### ISSUE THREAD

**Issue by Discostu36** [nextcloud/android#10729]
### Is your feature request related to a problem? Please describe.
When I am renaming a file, I usually use words in my language or names that are in my virtual keyboard dictionary. But I don't get any spelling suggestions when editing a file name, so I have to type them all letter by letter.

### Describe the solution you'd like
The file renaming input field should be configured that the keyboard does show spelling suggestions.

### Describe alternatives you've considered
I can't think of any.

**Comment by joshtrichards** [comment]
Hi @Discostu36 - This works for today - at least in v3.26.0. Can you confirm it's still not working for you?

**Comment by Discostu36** [comment]
No, I still don't get spelling suggestions when changing a file name.

![Screenshot_20231012-073031.png](https://github.com/nextcloud/android/assets/30999326/fc1122df-8271-4683-b0d9-3dee5d48e7af)

But I do get them when creating a new text file.

![Screenshot_20231012-072956.png](https://github.com/nextcloud/android/assets/30999326/78917548-3a60-4948-bcac-d918b0b13fb5)

Android 13
Fairphone 4
AnysoftKeyboard
